manifests/mod/dav_svn.pp

```diff
@@ -13,7 +13,7 @@
 
   ::apache::mod { 'dav_svn': }
 
-  if $::osfamily == 'Debian' and ($::operatingsystemmajrelease != '6' and $::operatingsystemmajrelease != '10.04' and $::operatingsystemrelease != '10.04' and $::operatingsystemmajrelease != '16.04') {
+  if $::osfamily == 'Debian' and ($::operatingsystemmajrelease != '6' and $::operatingsystemmajrelease != '10.04' and $::operatingsystemrelease != '10.04' and $::operatingsystemmajrelease != '16.04' and $::operatingsystemmajrelease != '9') {
```

Comment:
How about: 
```puppet
if $::osfamily == 'Debian' and $::operatingsystemrelease != '10.04' and ! ($::operatingsystemmajrelease in ['6', '10.04', '16.04', '9']) {
```

And maybe just drop 10.04 here :)

Reply:
@ekohl Updated :). Considered removing Deb 6 as well but as it's not as out of date decided to leave for now.

Reply:
Slightly easier to read now :)